src/backend/aspen/api/views/groups.py

```diff
@@ -134,7 +134,11 @@ async def invite_group_members(
         try:
             if success:
                 auth0_client.invite_member(
-                    organization["id"], client_id, user.name, email, "member"
+                    organization["id"],
+                    client_id,
+                    user.name,
+                    email,
+                    group_invitation_request.role,
```

Comment:
Send the role name that we get from the `group invite` cli to Auth0 so new users are created with a role.